src/main/java/com/jcabi/github/RtPull.java

```diff
@@ -159,6 +161,42 @@ public void merge(
     }
 
     @Override
+    public MergeState merge(
```

Comment:
@sonOfRa These two ```merge``` operation do almost the same. Can we extract some common part to the private method to reduce code duplication?
Something like this:
```java
private RestResponse merge(...)
```